pages/public_cloud/containers_orchestration/managed_kubernetes/Octavia_Tutorial_Add_Proxy_Protocol/guide.en-gb.md

```diff
@@ -0,0 +1,194 @@
+---
+title: 'Obtaining Source IP in Octavia'
+excerpt: 'Learn how to configure Octavia to obtain the client source IP in the context of OVHcloud without using Kubernetes CCM.'
+updated: 2024-08-05
+---
+
+## Objective
+
+**This guide explains how to configure Octavia to obtain the client source IP in the context of OVHcloud without using Kubernetes CCM.**
+
+## Requirements
+
+- Access to an OpenStack environment on OVHcloud with Octavia configured.
+- Administrative rights to create and modify LoadBalancer services.
+- A backend server ready to be used.
+
+## Instructions
+
+### Step 1: Configure the Backend Server on OVHcloud
+
+Create an instance on OVHcloud and install a web server that will serve as the backend. For example, we will use an NGINX server.
+
+#### 1. Create an instance on OVHcloud
+   
+Access your [OVHcloud Control Panel](/links/manager), go to the "Public Cloud" section and create a new instance. For detailed instructions, see the official documentation: [Creating an instance on OVHcloud](https://docs.ovh.com/en/public-cloud/create-vm/).
+
+#### 2. Install NGINX on the instance
+
+Connect to the instance via SSH
+   
+Update packages and install NGINX:
+
+```bash
+sudo dnf install -y nginx
```

Comment:
Check if this comand is working on all of the compute instances OS.
If not, precise the command for the OS.